Dockerfile

```diff
@@ -9,6 +9,12 @@ ENV PATH /root/.local/bin:$PATH
 RUN mkdir -p /app/user
 WORKDIR /app/user
 
+# Add a /app/GIT_HEAD_REF file with the git commit SHA that is currently
+# deployed
+COPY .git .git
```

Comment:
Do we need to copy in the whole directory? Could we copy only the `.git/HEAD` file we're looking for?